rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/stats/generateModel.groovy

```diff
@@ -0,0 +1,79 @@
+package org.jboss.windup.reporting.stats;
```

Comment:
At the least, this would belong in "scripts" or something rather than directly alongside the source code. And it will need to be documented.